src/java/org/apache/nutch/crawl/CrawlDb.java

```diff
@@ -236,10 +237,10 @@ public int run(String[] args) throws Exception {
    * Used for Nutch REST service
    */
   @Override
-  public Map<String, Object> run(Map<String, String> args, String crawlId) throws Exception {
+  public Map<String, Object> run(Map<String, Object> args, String crawlId) throws Exception {
```

Comment:
Thanks @sujen1412 for considering [nutch-python](https://github.com/chrismattmann/nutch-python) with this.